### PR TITLE
Android: ConfirmRunnable to RunRunnable with optional confirmation

### PR DESCRIPTION
**Pull request by Ebola16**
Some future runnable calls may not need confirmation.

### CODE REVIEW

**Comment by Ebola16** [comment]
And I changed the dialog buttons to OK / Cancel to make them more applicable for future uses.

File: Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/features/settings/model/view/RunRunnable.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+import org.dolphinemu.dolphinemu.features.settings.model.AbstractSetting;
+
+public final class RunRunnable extends SettingsItem
+{
+  private final boolean mConfirmRunnable;
+  private final int mAlertText;
+  private final int mToastTextAfterRun;
+  private final Runnable mRunnable;
+
+  public RunRunnable(int titleId, int descriptionId, boolean confirmRunnable, int alertText,
```
Comment (JosJuice) [inline Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/features/settings/model/view/RunRunnable.java:12]: What do you think about removing the `confirmRunnable` variable and instead making the code check `alertText != 0`, like you've done with `toastTextAfterRun`?

File: Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/features/settings/model/view/RunRunnable.java [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+  public Runnable getRunnable()
+  {
+    return mRunnable;
+  }
+
+  @Override
+  public int getType()
+  {
+    return TYPE_CONFIRM_RUNNABLE;
```
Comment (JosJuice) [inline Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/features/settings/model/view/RunRunnable.java:45]: The name of this constant should be changed as well.

**Review by JosJuice — COMMENTED** [review]
Hmm, I wonder why git is showing the files as entirely replaced instead of renamed and slightly edited... Oh well.

File: Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/features/settings/model/view/RunRunnable.java (hunk 1 above)
Comment (Ebola16) [reply Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/features/settings/model/view/RunRunnable.java]: Good idea. I used `alertTextID > 0` to stay consistent with the other ViewHolders.

**Review by JosJuice — APPROVED** [review]
Huh, now one of the two files shows up as renamed...? LGTM regardless.

**Comment by Ebola16** [comment]
I'm guessing git doesn't consider a file renamed if methods are modified? That's the most striking difference between the files in question.

**Pull request merged**